### DISCUSSION TITLE
can be deleted

### DISCUSSION THREAD

**Discussion by oliverk71** [mcguirepr89/BirdNET-Pi#1169] [Q&A]
I am running birdnet-pi on a Raspberry Pi 3B (not 3B+). However, it works fine in general. But after a couple of days I cannot reach it anymore. Until I plug it out and plug it in, again. This is really annoying. 

Another problem is the weekly report, which should be sent to my Mastodon account. Last time I got this report, was on 26th of February. 

Idk if someone had similar problems. 

Now I am up to a point, where I want to solve the problem forever. If I cannot, I will put it down and use the pi for something else. I would prefer the first alternative but do not have the time to care much about it.

**Comment by Vollpflock** [comment]
I have the same issue with a PI 4 (4 GB). However, I think it's because I am using an HHD. Will replace it with an SSD to test this.

**Reply by arne1921KF** [reply]
Just FTR, might be solved at [this](https://github.com/Nachtzuster/BirdNET-Pi/discussions) fork.
